linux/src/socket_transport_linux.c

```diff
@@ -581,32 +581,36 @@ int socket_transport_listen(SOCKET_TRANSPORT_HANDLE socket_transport, uint16_t p
     return result;
 }
 
-SOCKET_TRANSPORT_HANDLE socket_transport_accept(SOCKET_TRANSPORT_HANDLE socket_transport)
+SOCKET_ACCEPT_RESULT socket_transport_accept(SOCKET_TRANSPORT_HANDLE socket_transport, SOCKET_TRANSPORT_HANDLE* accepting_socket)
 {
     SOCKET_TRANSPORT* result;
-    // Codes_SOCKET_TRANSPORT_LINUX_11_069: [ If socket_transport is NULL, socket_transport_accept shall fail and return NULL. ]
+    SOCKET_ACCEPT_RESULT aresult;
```

Comment:
>aresult [](http://example.com/codeflow?start=25&length=7)

That's a funny looking variable, you can change this to just result since it's result.